sdk/storage/azblob/zt_client_provided_key_test.go

```diff
@@ -22,31 +25,37 @@ Azure Storage supports following operations support of sending customer-provided
 Put Blob, Put Block List, Put Block, Put Block from URL, Put Page, Put Page from URL, Append Block,
 Set Blob Properties, Set Blob Metadata, Get Blob, Get Blob Properties, Get Blob Metadata, Snapshot Blob.
 */
-var testEncryptedKey = "MDEyMzQ1NjcwMTIzNDU2NzAxMjM0NTY3MDEyMzQ1Njc="
-var testEncryptedHash = "3QFFFpRA5+XANHqwwbT4yXDmrT/2JaLt/FKHjzhOdoE="
-var testEncryptionAlgorithm = "AES256"
-var testCPKByValue = CpkInfo{
-	EncryptionKey:       &testEncryptedKey,
-	EncryptionKeySHA256: &testEncryptedHash,
-	EncryptionAlgorithm: &testEncryptionAlgorithm,
+func getCPKEncryptionKey() *CpkInfo {
+	return &CpkInfo{
+		EncryptionKey:       to.StringPtr("MDEyMzQ1NjcwMTIzNDU2NzAxMjM0NTY3MDEyMzQ1Njc="),
```

Comment:
Are these valid keys? if so, they should not be stored in plain text and should be read in via environment variables

Reply:
Looks like it's just "01234567012345670123456701234567". 

To make it easier on future reviews you might just want to base 64 encode that directly in your test, so it's obvious what it's coming from.